requirements.txt

```diff
@@ -0,0 +1,16 @@
+pyyaml >= 5.1
```

Comment:
Unfortunate that we need these twice: here and in `setup.cfg`.  Is this the standard way to maintain these?

Reply:
I will admit that @jonathansick tried to explain to me once why we have to duplicate requirements but I haven't really grokked it yet. You can't do "pip install requirements from setup.cfg" so a separate file gives you that.  setuptools doesn't know about requirements.txt so ignores them and uses the ones from setup.cfg.

Reply:
I don't understand the strategy here. If you've got a `setup.cfg`, then you shouldn't need this `requirements.txt` file at all. What is the `requirements.txt` file being used for?

Reply:
Ok. I thought we had a whole conversation about this back when you were here.  I was following: https://packaging.python.org/discussions/install-requires-vs-requirements/  to a certain extent except I wasn't really doing the "minimal in setup.cfg and maximal in requirements.txt approach".

I'm pretty sure you can't ask pip to install requirements specified in a setup.cfg so you absolutely need the requirements.txt for that (and you need to somehow install setuptools_cpp when building sphgeom before you even run setup.py since that's required to configure the C++ build). On the other hand setuptools doesn't look at requirements.txt and I don't see situations where people try to read the requirements.txt into their setup.py execution. Also requirements.txt lets people preinstall dependencies using pip before they try to install the package itself.  It seems like an inconsistent mess at first glance.